constants/constants.go

```diff
@@ -27,6 +27,7 @@ const (
 	DefaultImageRepository = "quay.io/coreos/etcd"
 	DefaultCertificateDir  = "/etc/etcd/pki"
 
+	// TODO: replace default name
```

Comment:
That's going to be a breaking change, so it might not be easy.  But what are you thinking of here?  Supporting multiple instances of etcd, or something else?